manifests/o/Oracle/JDK/21/21.0.1/Oracle.JDK.21.installer.yaml

```diff
@@ -0,0 +1,23 @@
+# Created using wingetcreate 1.5.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.5.0.schema.json
+
+PackageIdentifier: Oracle.JDK.21
+PackageVersion: 21.0.1
+InstallerLocale: en-US
+Platform:
+- Windows.Desktop
+InstallerType: wix
+InstallModes:
+- interactive
+- silent
+UpgradeBehavior: uninstallPrevious
+ProductCode: '{75B0E1AE-DC20-5AC0-A358-61B0256DADBE}'
+AppsAndFeaturesEntries:
+- DisplayName: Java(TM) SE Development Kit 21 (64-bit)
+Installers:
+- Architecture: x64
+  InstallerUrl: https://download.oracle.com/java/21/latest/jdk-21_windows-x64_bin.msi

```

Comment:
Since this is a vanity URL, the previous version should be removed (can be part of this PR or a separate one)